python/ml4ir/applications/ranking/tests/test_feature_layer.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+from ml4ir.applications.ranking.tests.test_base import RankingTestBase
+from ml4ir.base.features import preprocessing
+from ml4ir.base.features.feature_fns import categorical as categorical_fns
+from ml4ir.base.features.feature_fns import sequence as sequence_fns
+from ml4ir.base.config.keys import SequenceExampleTypeKey
+
+
+# class RankingModelTest(RankingTestBase):
```

Comment:
Do you plan on uncommenting this? What's the status?

Reply:
Looks like this is "outdated" code. I have updated this module with the the python unit tests for the feature layer.